lib/matplotlib/tests/test_bezier.py

```diff
@@ -0,0 +1,25 @@
+from matplotlib.tests.test_path import _test_curves
+
+import numpy as np
+import pytest
+import scipy.integrate
+
+
+# get several curves to test our code on by borrowing the tests cases used in
+# `~.tests.test_path`. get last path element ([-1]) and curve, not code ([0])
+_test_curves = [list(tc.path.iter_bezier())[-1][0] for tc in _test_curves]
+
+
+def _integral_arc_area(B):
+    """(Signed) area swept out by ray from origin to curve."""
+    dB = B.differentiate(B)
+    def integrand(t):
+        dr = dB(t).T
+        n = np.array([dr[1], -dr[0]])
```

Comment:
This is a the 2D case of the more general algorithm for finding a normal vector to an arbitary non-zero vector, see https://math.stackexchange.com/a/211195.